src/gt4py/next/ffront/fbuiltins.py

```diff
@@ -291,13 +319,13 @@ def impl(
 #  guidelines for decision.
 @dataclasses.dataclass(frozen=True)
 class FieldOffset(runtime.Offset):
-    source: Dimension
-    target: tuple[Dimension] | tuple[Dimension, Dimension]
+    source: common.Dimension
+    target: tuple[common.Dimension] | tuple[common.Dimension, common.Dimension]
     connectivity: Optional[Any] = None  # TODO
 
     def __post_init__(self):
-        if len(self.target) == 2 and self.target[1].kind != DimensionKind.LOCAL:
-            raise ValueError("Second dimension in offset must be a local dimension.")
+        if len(self.target) == 2 and self.target[1].kind != common.DimensionKind.LOCAL:
+            raise ValueError("Second Dimension in offset must be a local Dimension.")
```

Comment:
```suggestion
            raise ValueError("Second Dimension in Offset must be a local Dimension.")
```
If you insist on this utterly ugly change, go the full mile :-P Not sure why we would want this, here the concept of a dimension was meant, which looked fine to me.

Reply:
that was a find/replace side-effect which apparently I undid only partially...